recipe/meta.yaml

```diff
@@ -7,33 +9,39 @@ package:
 source:
   fn: python-snappy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/python-snappy/python-snappy-{{ version }}.tar.gz
-  md5: 0c06bbed70a8390b55a60f5ee79a27d2
+  sha256: {{ sha256 }}
+  patches:
+    - msvc-inline.patch  # [win and py27]
```

Comment:
Would actually just apply this patch across the board. This would serve as a good test that it doesn't mess up other Python versions on Windows and works ok on other OSes. As it is all preprocessor statements, this should have no negative effects outside of a tiny bit more compilation time. By doing this, it should also make it easier for upstream to accept the change (and us to rest easy) knowing that it works everywhere just fine. Not that I would expect otherwise. 😉